docs/rest/src/concepts.rst

```diff
@@ -83,6 +83,20 @@ understanding the requirements for integration with WOPI clients such as |wac| a
 
         ..  include:: /_fragments/access_token_handling_warning.rst
 
+        **The anonymous access token**
+
+        WOPI operations always require an access token value. However, there are situations where a WOPI host may wish
+        to allow some operations to be anonymously callable. To accomodate such cases, the string ``anonymous``
+        is a reserved access token value in WOPI. Both hosts and clients can use this value in cases where an access
+        token is required, but the host does not actually care to verify the token.
```

Comment:
Good questions. I think the expected behavior would be a 401 or 404 if the access token was missing. Arguably a 400 would be appropriate too, but we don't currently use 400 anywhere in WOPI (don't quote me on that; I haven't done an exhaustive search but I can't think of any examples off the top of my head).

Since I'm closing this PR, I'm not going to make these changes, but this will be relevant if we bring this proposal back.